### DISCUSSION TITLE
Support for Redis Streams

### DISCUSSION THREAD

**Discussion by PyAntony** [smallrye/smallrye-reactive-messaging#2054] [Ideas]
Hi,

1) Is a connector for Redis Streams in the making?
2) Is there a standard way to implement custom connectors? Spring Cloud Stream offers a standard API to create custom `binder`s for example : https://docs.spring.io/spring-cloud-stream/docs/Brooklyn.RELEASE/reference/html/_binders.html. I know I can just create my own logic to hook into the reactive streams (using polling, queues, etc.), but I wonder if there is a set of interfaces that I can just implement (or is a SPI already in the making?).

**Comment by cescoffier** [comment]
You can find information about connectors at https://smallrye.io/smallrye-reactive-messaging/4.1.1/ and https://github.com/smallrye/smallrye-reactive-messaging/. 

The MicroProfile Reactive Messaging specification defines the SPI in question.
